after_write_hooks/sqs.rb

```diff
@@ -6,30 +6,33 @@
 
 module AfterWriteHooks
   class SQS
-    def self.run(config_file_path, s3_bucket, s3_path, sqs_prefix)
-      event_type = s3_path.split('/')[1]
-      queue_configs = self.config(config_file_path)['queues']
-      default_config = queue_configs['default']
-      configs = queue_configs[event_type] || default_config
+    QUEUE_SUFFIX = 'redshift_etl'
 
-      configs.each do |queue_config|
-        # like "someevent-in-demo" or "someevent-in-production"
-        queue_name = queue_config['name'] || "#{event_type}-in-#{s3_bucket.split('-').last}"
-        queue_name = sqs_prefix + queue_name
+    # @param [string] config_file_path
+    #   The config file contains the list of SQS queues
+    #   Example: "/etc/td-agent/sqs.conf"
+    # @param [string] s3_bucket
+    #   Example: "change-fluentd-production"
+    # @param [string] s3_path
+    #   The key in S3
+    #   Example: "events/petition_view/production-change_main_fluentd_hub-00/2015/05/28/21_0.json"
+    def self.run(config_file_path, s3_bucket, s3_path)
+      queue_config = YAML.load_file(config_file_path)
+      environment = s3_bucket.split('-').last  # demo, production, staging, etc.
+      queue_name = "#{environment}-#{QUEUE_SUFFIX}" # production-redshift_etl, staging-redshift_etl, etc.
+      event_name = s3_path.split('/')[1] # petition_view, share_petition, etc.
 
-        sqs = AWS::SQS.new(
-          access_key_id: queue_config['access_key_id'],
-          secret_access_key: queue_config['secret_access_key'],
-          region: queue_config['region']
-        )
-        message = {s3_bucket: s3_bucket, s3_path: s3_path}
-        queue = sqs.queues.named(queue_name)
-        queue.send_message(message.to_json)
-      end
-    end
-
-    def self.config(config_file_path)
-      @config ||= YAML.load_file(config_file_path)
+      sqs = AWS::SQS.new(
+        access_key_id: queue_config['access_key_id'],
+        secret_access_key: queue_config['secret_access_key'],
+        region: queue_config['region']
+      )
+      queue = sqs.queues.named(queue_name)
+      queue.send_message({
+        s3_bucket: s3_bucket,
+        s3_path: s3_path,
+        event_name: event_name
```

Comment:
We're now including the event name in the SQS message so we don't have to parse it out of the `s3_path`